### PR TITLE
WIP: Use Paket Pack

### PR DESCRIPTION
**Pull request by forki**
This tries to make #170 work

### CODE REVIEW

File: build.fsx [hunk 1]
```diff
@@ -305,10 +297,12 @@ Target "CI" DoNothing
...
-  ==> "NuGet"
+  ==> "PaketPack" 
+  ==> "PaketPush"
   ==> "Release"
 
-"GenerateDocs" ==> "CI"
-"NuGet" ==> "CI"
+"GenerateDocs"
+  ==> "PaketPack" 
+  ==> "CI"
```
Comment (kurtschelfthout) [inline build.fsx]: Not a big deal but I like to have separate "generate nuget packages" and "generate documentation" targets, i.e. if I'm checking the packaging I don't necessarily want to generate the docs.

File: build.fsx [hunk 2]
```diff
@@ -198,24 +196,19 @@ Target "SourceLink" (fun _ ->
...
+          OutputPath = "../../bin/PaketPack"
+          Version = buildVersion
+          ReleaseNotes = toLines release.Notes
+      })
+)
+
+Target "PaketPush" (fun _ ->
+    Paket.Push (fun p -> 
+        { p with 
+            WorkingDir = "../../bin/PaketPack" })
```
Comment (kurtschelfthout) [inline build.fsx]: Does this work the same way with regards to the nugetkey? I.e. clearly the push won't work unless it knows my key, but do I still set is as an env variable in my (not checked in) release script?

**Comment by kurtschelfthout** [comment]
This is looking really good. Thanks a bunch!

File: build.fsx (hunk 2 above)
Comment (forki) [inline build.fsx]: Yes it uses same conventions as before

**Pull request merged**